dlc-manager/src/sub_channel_manager.rs

```diff
@@ -2774,39 +2913,50 @@ where
                                                 .offered_contract
                                                 .id,
                                             temporary_channel_id: dlc_channel.temporary_channel_id,
-                                            party_points: dlc_channel.own_points,
-                                            per_update_point: dlc_channel.own_per_update_point,
-                                            offer_per_update_seed: channel.per_split_seed,
+                                            party_points: dlc_channel.counter_points,
+                                            per_update_point: dlc_channel.counter_per_update_point,
```

Comment:
Because the code used to be for the offer party side, but now it's in the accept party side.

Reply:
This is slightly confusing still. I have two questions:

- Are we interpreting these fields differently depending on the value of `is_offer_party`? For example, if `is_offer_party == false`, do we take `party_points` to mean the offerer's points which are therefore the counterparty's?
- What does the `counter_party` field below mean if these fields depend on `is_offer_party`. It looks like we also need to flip that to be consistent with this change.

It might help to take an absolute perspective in this message, with fields belonging to "sender" and "receiver" rather than "ours" and "theirs". The `is_offer_party` field would remain inherently relative as constructed, although we could instead say `pubkey_offer_party` so that each side can compare against their own pubkey. Not sure if it's worth it though.

Reply:
> Are we interpreting these fields differently depending on the value of is_offer_party? For example, if is_offer_party == false, do we take party_points to mean the offerer's points which are therefore the counterparty's?

Yes

> What does the counter_party field below mean if these fields depend on is_offer_party. It looks like we also need to flip that to be consistent with this change.

The `counter_party` field is actually just the counter party pubkey, regardless of the `is_offer_party` value.

> It might help to take an absolute perspective in this message, with fields belonging to "sender" and "receiver" rather than "ours" and "theirs".

So pretty much all fields are the ones of the "sender", there is really nothing about the "receiver".

> The is_offer_party field would remain inherently relative as constructed, although we could instead say pubkey_offer_party so that each side can compare against their own pubkey. Not sure if it's worth it though.

If you find it clearer or easier I'm ok with such a refactor, but maybe not as part of this PR :). I agree it might prevent some errors (where the field is set incorrectly).

Reply:
Thanks for clarifying!

> If you find it clearer or easier I'm ok with such a refactor, but maybe not as part of this PR :). I agree it might prevent some errors (where the field is set incorrectly).

I agree that we shouldn't do it with this PR. I might give it a go in the future 😄